<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>486</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MIDI Timing Master</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>486</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionMIDI_Settings"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAbout"/>
  </widget>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="MidiTimingMaster.qrc">
     <normaloff>:/Sequencer/icons/about.png</normaloff>:/Sequencer/icons/about.png</iconset>
   </property>
   <property name="text">
    <string>About...</string>
   </property>
  </action>
  <action name="actionMIDI_Settings">
   <property name="icon">
    <iconset resource="MidiTimingMaster.qrc">
     <normaloff>:/Sequencer/icons/actionSettings.png</normaloff>:/Sequencer/icons/actionSettings.png</iconset>
   </property>
   <property name="text">
    <string>MIDI Settings</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="MidiTimingMaster.qrc">
     <normaloff>:/Sequencer/icons/actionExit.png</normaloff>:/Sequencer/icons/actionExit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="MidiTimingMaster.qrc"/>
 </resources>
 <connections/>
</ui>
